Core/Graphics/Source/FrameBuffer.cpp

```diff
@@ -108,6 +108,8 @@ namespace Babylon
         // if no other draw calls are submitted to the view.
         if (doTouch)
         {
+            // discard everything or touch call will use what is bound.
+            encoder->discard();
```

Comment:
I don't think this is necessary anymore because we are discarding everything except for bindings now.